lib/parsers.js

```diff
@@ -106,20 +124,20 @@ function parse(options) {
     var parseOptions = {
         customTags: customTags
     };
-    var originalParser = parsersDb[ext];
-    var parsers = [].concat(originalParser.parserName);
+    var originalParser = customParsers[ext] || parsersDb[ext];
+    var parserNames = [].concat(originalParser.parserName);
 
     var includedFiles = originalParser.includedFiles || [];
     if (withInlineFiles) {
         includedFiles.forEach(function (ext) {
-            parsers.push(parsersDb[ext].parserName);
+            parserNames.push(parsersDb[ext].parserName);
```

Comment:
I think included files should also adapt to available `customParsers` if they defined